code/game/objects/items/weapons/melee/misc_vr.dm

```diff
@@ -0,0 +1,16 @@
+/obj/item/weapon/melee/rapier
+	name = "rapier"
+	desc = "A gleaming steel blade with a gold handguard and inlayed with an outstanding red gem."
+	icon = 'icons/obj/weapons_vr.dmi'
+	icon_state = "rapier"
+	item_icons = list(
+		slot_l_hand_str = 'icons/mob/items/lefthand_melee_vr.dmi',
+		slot_r_hand_str = 'icons/mob/items/righthand_melee_vr.dmi',
+		)
+	force = 25
```

Comment:
This probably should have force in line with something that's meant to be ceremonial.